app/src/main/java/com/willowtree/vocable/settings/EditKeyboardFragment.kt

```diff
@@ -22,11 +22,21 @@ class EditKeyboardFragment : BaseFragment() {
 
     companion object {
         private const val KEY_PHRASE = "KEY_PHRASE"
+        private const val KEY_IS_EDITING = "KEY_IS_EDITING"
 
         fun newInstance(phrase: Phrase): EditKeyboardFragment {
             return EditKeyboardFragment().apply {
                 arguments = Bundle().apply {
                     putParcelable(KEY_PHRASE, phrase)
+                    putBoolean(KEY_IS_EDITING, true)
+                }
+            }
+        }
+
+        fun newInstance(isEditing: Boolean): EditKeyboardFragment {
+            return EditKeyboardFragment().apply {
```

Comment:
Since y'all are using the `ktx-core` dependency, you can take advantage of [bundleOf](https://developer.android.com/reference/kotlin/androidx/core/os/package-summary#bundleOf(kotlin.Pair)) (so this would be `arguments = bundleOf(KEY_IS_EDITING to true)`

Since this is a single statement technically, you can also single line it (`fun newInstance(isEditing: Boolean) = EditKeyboardFragment().apply ....`)